src/server/api/controllers/modules.controller.crop-stages-endpoint.js

```diff
@@ -0,0 +1,24 @@
+const knex = require('../../config/db');
+
+const getCropStages = async () => {
+  try {
+    const startDate = await knex('batches')
+      .select('seeding_date')
+      .where({ id: '31' });
+    const stages = await knex('crop_stages').select(
+      'crop_stages.name',
+      'crop_stages.duration',
+    );
+    const date = new Date();
+    return {
+      startDate: startDate[0].seeding_date,
```

Comment:
Instead of this, you should query `[0]` to get the first occurrence you can use `.first()` in the query for `batches`.

Also this will not behave correctly you might not get the same result everytime. Consider ordering by date or similar to make sure you get the correct value.

Reply:
what date should we consider  ?  .. i mean the last batch entered (if we have multiple batches) ? or do we have only one batch at a time ?